viewing_party/party.py

```diff
@@ -1,23 +1,162 @@
 # ------------- WAVE 1 --------------------
+from platform import libc_ver

```

Comment:
Looks like this `import` statement never got used. I know there's a VS Code extension that will add these automatically that often causes it by offering suggestions that create these imports but are most likely not what you want. I would recommend disabling it, Ansel describes how to in [this Slack post](https://adad18.slack.com/archives/C03QJHJDW2V/p1663446293671619?thread_ts=1663446288.654079&cid=C03QJHJDW2V).

Failing that, make sure to remove these types of statements if they never get used just as a style practice.